clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemCore.java

```diff
@@ -153,4 +167,16 @@ boolean rename(TachyonFile src, TachyonURI dst, RenameOptions options) throws IO
    * @throws TachyonException if a Tachyon exception occurs
    */
   void setState(TachyonFile file, SetStateOptions options) throws IOException, TachyonException;
+
+  /**
+   * Unmounts a UFS subtree from Tachyon.
+   *
+   * @param tachyonPath a Tachyon path
```

Comment:
not particularly happy with the javadoc here, e.g., what is the relationship between the tachyonPath and the UFS subtree. We may want to explain this.